docs/overview.rst

```diff
@@ -264,81 +246,27 @@ Admin UI
 Next Steps
 ~~~~~~~~~~
 
-* **Connect to your cluster**: Click the "view code snippets" link to
-  see a list of snippets for the available clients and libraries. These
-  include: CLI clients, Java, JavaScript, PHP, Python, and Ruby.
-
-* **Demo data**: Clicking the "import the demo data" will show you two
-  snippets that can be used in the SQL Console of your Admin UI. The first
-  snippet will create a "nyc_taxi" table, and the second one will populate it with our 
-  demo dataset.
+* **Import Data**: Import some data into your cluster using our data import 
+  tool.
 
-* **Backups**: The "see my backups" will take you to the Backups tab, where 
-  you can see all your backups.
+* **See my backups**: CrateDB Cloud clusters can now be cloned to a new 
+  cluster from any backup.
```

Comment:
I think this was a a pretty good explanation previously? Maybe add the clone sentence at the end?

```suggestion
* **See my backups**: The "see my backups" will take you to the Backups tab, where 
  you can see all your backups. CrateDB Cloud clusters can now be cloned to a new 
  cluster from any backup.
```